Database/DatabaseConnection.py

```diff
@@ -7,31 +7,38 @@
 Hao Wei Huang
 """
 
-import psycopg2
 import logging
 import sys
 import os
+import oracledb
 
 
 class DatabaseConnection(object):
-    def __init__(self, user="postgres", password="1234", host="127.0.0.1", port="5432", database="postgres",
+    def __init__(self, user="SYSTEM", password="password", dsn = "db:1521/ORCLCDB",
                  course_table="course_info", prereqs_table="prereqs", antireqs_table="antireqs",
                  requirements_table="requirements", communications_table="communications",
                  breadth_table="breadth_table"):
-        if os.getenv("UWPATH_ENVIRONMENT") is not None and os.getenv("UWPATH_ENVIRONMENT") == "docker":
-            host = "db"
         if os.getenv("DB_PASS") is not None:
             password = os.getenv("DB_PASS")
         if os.getenv("DB_USER") is not None:
             user = os.getenv("DB_USER")
-        if os.getenv("DB_NAME") is not None:
-            database = os.getenv("DB_NAME")
-        if os.getenv("DB_HOST") is not None:
-            host = os.getenv("DB_HOST")
-        if os.getenv("DB_PORT") is not None:
-            port = os.getenv("DB_PORT")
+        if os.getenv("ORACLE_DSN") is not None:
```

Comment:
Make sure you have this thing saved somewhere secure if needed to pull it up again :)

Reply:
not sure what you mean here. We have the values, but they will be injected when we build the pipeline

Reply:
that sounds good, just make sure the value isn't in the public repo somewhere for security reasons

Reply:
Of course. Sounds good